File: Source/Hafner.Compatibility.MetaPackage.CompileTests.VB/Hafner.Compatibility.GuidAttribute/Interface1.vb
'GuidAttribute on interface level

Imports System.Runtime.InteropServices

Namespace GuidAttributeTests

    <Guid("74EF5210-4624-4E3C-88F3-2B2E4A2CE3D0")>
    Interface IInterface1

        ReadOnly Property Name As String

    End Interface

End Namespace

File: Source/Hafner.Compatibility.MetaPackage.CompileTests.CS/Hafner.Compatibility.GuidAttribute/Interface1.cs
//GuidAttribute on interface level

namespace Hafner.Compatibility.CompileTest.CS.GuidAttribute;

using System.Runtime.InteropServices;

[Guid("74EF5210-4624-4E3C-88F3-2B2E4A2CE3D0")]
public interface IInterface1 {

    string Name { get; }

}
